java/sage/Agent.java

```diff
@@ -474,7 +483,7 @@ void unbully(int weakerID)
       {
         Airing a = (Airing) allAirs[i];
         if (a == null) continue;
-        if (followsTrend(a, true, sbCache))
+        if (followsTrend(a, true, sbCache, false, true))
```

Comment:
This should pass through ignoreDisabled as the last value (I understand it'll have the same functionality, but it just looks more correct).